packages/suite/src/components/suite/Labeling/components/Wallet/index.tsx

```diff
@@ -1,26 +1,28 @@
 import React from 'react';
 import { Translation } from '@suite-components';
 import { TrezorDevice } from '@suite-types';
-import messages from '@suite/support/messages';
 
 interface Props {
     device: TrezorDevice;
     useDeviceLabel?: boolean;
 }
 
-const Wallet = (props: Props) => {
+export const WalletLabeling = (props: Props) => {
```

Comment:
Why change it to a named export?

Reply:
I was changing Wallet to Wallet labeling as it is inconsitent with other labeling components. And I somehow started to prefer named over default exports, but I am reverting this for now for the sake of consistency.